net-nntp/sabnzbd/sabnzbd-3.6.1.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..11} )
+PYTHON_REQ_USE="sqlite"
+
+inherit python-single-r1 systemd
+
+MY_PV="${PV/_rc/RC}"
+MY_PV="${MY_PV//_pre*}"
+
+MY_P="${PN/sab/SAB}-${MY_PV}"
+
+DESCRIPTION="Binary newsgrabber with web-interface"
+HOMEPAGE="https://sabnzbd.org/"
+SRC_URI="https://github.com/sabnzbd/sabnzbd/releases/download/${MY_PV}/${MY_P}-src.tar.gz"
+
+# Sabnzbd is GPL-2 but bundles software with the following licenses.
+LICENSE="GPL-2 BSD LGPL-2 MIT BSD-1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+7za +rar unzip test"
+RESTRICT="!test? ( test )"
+
+# Is this still correct? The article has disappeared from the wiki!
+# Sabnzbd is installed to /usr/share/ as upstream makes it clear they should not
+# be in python's sitedir.  See: https://sabnzbd.org/wiki/advanced/unix-packaging
+
+COMMON_DEPS="
+	acct-user/sabnzbd
+	acct-group/sabnzbd
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cheetah3[${PYTHON_USEDEP}]
+		dev-python/cherrypy[${PYTHON_USEDEP}]
+		dev-python/configobj[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/feedparser-6.0.10[${PYTHON_USEDEP}]
+		>=dev-python/guessit-3.1.0[${PYTHON_USEDEP}]
+		dev-python/notify2[${PYTHON_USEDEP}]
+		dev-python/portend[${PYTHON_USEDEP}]
+		dev-python/puremagic[${PYTHON_USEDEP}]
+		>=dev-python/sabyenc-5.4.2[${PYTHON_USEDEP}]
+	')
+"
+
+DEPEND="${COMMON_DEPS}"
+
+RDEPEND="
+	${COMMON_DEPS}
+	>=app-arch/par2cmdline-0.4
+	net-misc/wget
+	7za? ( app-arch/p7zip )
+	rar? ( || ( app-arch/unrar app-arch/rar ) )
+	unzip? ( >=app-arch/unzip-5.5.2 )
+"
+
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flaky[${PYTHON_USEDEP}]
+			dev-python/pkginfo[${PYTHON_USEDEP}]
+			dev-python/pyfakefs[${PYTHON_USEDEP}]
+			dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+			dev-python/pytest-httpserver[${PYTHON_USEDEP}]
+			dev-python/selenium[${PYTHON_USEDEP}]
+			dev-python/xmltodict[${PYTHON_USEDEP}]
+		')
+		www-apps/chromedriver-bin
+	)
+"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	MY_HOMEDIR="/var/lib/${PN}"
```

Comment:
I wonder what the function for this is. If you know and it makes sense, please add a comment above explaining it.

Reply:
I got rid of the MY_HOMEDIR since it was only used once in postinstall. The rest is about changing image install directory. Does it need a comment? Other packages use it and don't have any.